simpleui-server/src/sui_data.ts

```diff
@@ -493,7 +493,7 @@ export class SuiData {
 
         let attributes = '';
 
-        if (req.query !== {}) {
+        if (typeof req.query !== 'object') {
```

Comment:
Don't understand this change -- the former version seemed to be doing the test `is the req.query object empty?`, whereas this change makes the test `is req.query  NOT an object?`.

`Object.keys(req.query)` below can only work if `req.query` is an object.

Reply:
Just added a commit to fix this

Reply:
Maybe something to fix later.

Looking at it again, I think you have identified a real problem. I wonder if the correct query is to identify if it IS an object: `if (typeof req.query === 'object')`.  Ideally, the condition could be eliminated completely if the subsequent line works correctly with different possible values of `req.query`:  null, empty string, empty object, array.

Reply:
huh, good point! I know that if `req.query` is a non-empty string, `Object.keys` will output the indexes of the string which I dont think is desired behavior. I also noticed that `req.query !== {}` would always evaluate to true since js compares objects by reference instead of value. I can find a solution to this and add it soon